.travis.yml

```diff
@@ -48,6 +48,10 @@ before_script:
 script:
   - ./travis_configure.sh run_tests
 
+after_success:
+  - beaver docker build -f "docker/Dockerfile.ubuntu.xenial" .
+  - BEAVER_CODECOV_REPORTS="build/libdrizzle-redux-coverage.info" beaver codecov -X gcov -Z
```

Comment:
Is it OK to use this for all builds? For builds that already use `beaver` to run stuff inside docker, it shouldn't be necessary to build the docker image again.